static/js/components/LocationPicker.js

```diff
@@ -0,0 +1,75 @@
+// @flow
+/* global SETTINGS: false */
+import React from "react"
+import Places from "places.js"
+
+type Props = {
+  placeholder?: string,
+  onChange: Function,
+  onClear: Function,
+  options: Object,
+  initialLocation: ?string
+}
+
+export default class LocationPicker extends React.Component<Props> {
+  autocomplete: Object
+  autocompleteElem: ?Object
+
+  static defaultProps = {
+    placeholder:     "Location (city)",
+    initialLocation: "",
+    options:         {
+      type:     "city",
+      language: "en"
+    }
+  }
+
+  componentDidMount() {
+    const { onChange, onClear, options } = this.props
+    this.autocomplete = Places({
+      appId:     SETTINGS.algolia_appId,
+      apiKey:    SETTINGS.algolia_apiKey,
+      ...options,
+      container: this.autocompleteElem
+    })
+
+    this.autocomplete.on("change", onChange)
+    this.autocomplete.on("clear", onClear)
+  }
+
+  shouldComponentUpdate() {
+    return false
```

Comment:
Could you add a comment explaining why this is statically `false`? I can guess from the code, but I'm not sure I'm right or that we'll remember.

Reply:
I think I can actually remove this method altogether.